orchestra/static/dist/main.css

```diff
@@ -2345,6 +2345,8 @@ a.logo {
   .list-by-status .angular-ui-tree-node {
```

Comment:
unrelated to this PR: by default, does the `newTodoTaskId` get set to the current task's value, or will one expert's dropdown be set to another expert's `newTodoTaskId` by default?

Reply:
One expert's dropdown can be set to another expert's newTodoTaskId.